components/motor/gpio/motor_encoder.go

```diff
@@ -270,10 +268,10 @@ func (m *EncodedMotor) makeAdjustments(pos, lastPos float64, now, lastTime int64
 	}
 
 	if rpmDebug {
```

Comment:
This is never initialised now - initialize it as a global constant at the top of the file that is false

Reply:
it is on line 25 it just isn't a change so it didn't show up. that's sufficient?

Reply:
ah okay yeah!